basic/map.cpp

```diff
@@ -822,9 +822,9 @@ void ARegionList::MakeUWMaze(ARegionArray *pArr)
             ARegion *reg = pArr->GetRegion(x, y);
             if (!reg) continue;
 
-            for (int i=D_NORTH; i<= NDIRS; i++) {
+            for (int i=D_NORTH; i != NDIRS; i++) {
                 int count = 0;
-                for (int j=D_NORTH; j< NDIRS; j++)
+                for (int j=D_NORTH; j != NDIRS; j++)
                     if (reg->neighbors[j]) count++;
```

Comment:
I actually don't see the logical point to make this changes.  I *believe* that they are logically effectively equivalent and so the change feels stylistic rather than functional and while I do make stylistic changes at times, I don't particularly see the point of this one.

Reply:
If you can show me the case where this would cause some out of bounds error, I would be more willing to accept it, but I do not see it.